src/plugins/codeStyle/diagnosticMessages.ts

```diff
@@ -199,5 +201,19 @@ export const messages = {
         source: 'bslint',
         message: `${CS} File should follow Roku broadcast safe color cert requirement`,
         range
+    }),
+    noAssocarrayFieldType: (range: Range, severity: DiagnosticSeverity) => ({
+        message: `Using 'assocarray' type in component markup can result in inefficient copying of data during transfer to the render thread. Use 'node' type if possible for more efficient transfer of data from the task thread to the render thread`,
```

Comment:
This message and the one for array feels really verbose. This message is better served in the documentation for a rule instead of the error message itself. 

I think this message should be something more like this:

`"Do not use field type 'assocarray'"`

Reply:
Message was adjusted